docs/contributing-guide/1. getting-set-up.md

```diff
@@ -12,9 +12,9 @@
 
 ### Prerequisites
 
-- Ensure node is installed. [Node.js (16.19.0 and above)](https://nodejs.org/en/download/).
-- (Recommended) install vscode. [Visual Studio Code (Stable Build)](https://code.visualstudio.com/Download).
-- Install [rush](https://rushjs.io/), in a terminal of your choosing, run:
+- Install [Node.js (16.19.0 and above)](https://nodejs.org/en/download/).
```

Comment:
should we recommend developers use nvm instead and install 16.19.0. Perhaps we can move it to another spot if its too lengthy

Reply:
I think we should, certainly cleaner that way.

Reply:
But we don't have any `nvm` commands mentioned in this Contribution Guideline. I think it would be better to include them in the document where they are used.